index.html

```diff
@@ -2232,12 +2233,22 @@ <h5>Conclusion</h5>
 <section id="capability-single-custom-image" data-ucr-role="capability">
 <h4>Display a map using a single custom image as a map layer</h4>
 <p>
-  While most web maps often are often composed of tilesets, they may also consist of one of more layers that are just single images rendered on the fly (such as from a WMS), large images that a user pans across or vector images. These kinds of maps can be useful in cases where a broader spatial context is not as important or when the author is trying to make a specific point about a set of data. Custom image layers can also be used for representing complex data for a specific area that may not be available as a complete tileset, such terrain maps or foliage coverage images produced by a specific researcher.
+While most web maps often are often composed of tilesets, they may also
```

Comment:
"often are often", is the word often used too often here? :)

Reply:
Thank you @Malvoz! Fixed :)